src/System.Globalization/tests/NumberFormatInfo/NumberFormatInfoData.cs

```diff
@@ -6,47 +6,38 @@ namespace System.Globalization.Tests
 {
     internal static class NumberFormatInfoData
     {
-        public static int[] GetNumberGroupSizes(string localeName)
+        public static int[] UrINNumberGroupSizes()
         {
-            if (string.Equals(localeName, "en-US", StringComparison.OrdinalIgnoreCase))
+            if ((PlatformDetection.IsWindows && PlatformDetection.WindowsVersion >= 10)
+                ||
+                (PlatformDetection.IsOSX && PlatformDetection.OSXKernelVersion >= new Version(15, 0)))
             {
                 return new int[] { 3 };
             }
-            if (string.Equals(localeName, "ur-IN", StringComparison.OrdinalIgnoreCase))
+            else
             {
-                if ((PlatformDetection.IsWindows && PlatformDetection.WindowsVersion >= 10)
-                    ||
-                    (PlatformDetection.IsOSX && PlatformDetection.OSXKernelVersion >= new Version(15, 0)))
-                {
-                    return new int[] { 3 };
-                }
-                else
-                {
-                    return new int[] { 3, 2 };
-                }
+                return new int[] { 3, 2 };
             }
-
-            throw DateTimeFormatInfoData.GetCultureNotSupportedException(new CultureInfo(localeName));
         }
 
-        internal static string GetNegativeInfinitySymbol(string localeName)
+        internal static string GetPositiveInfinitySymbol(string localeName)
         {
             if (PlatformDetection.IsWindows && PlatformDetection.WindowsVersion < 10)
```

Comment:
I think it is better to format a float infinity number using the desired culture and use the result as the symbol you are looking for. I am just trying to avoid using hardcoded data again :-)